rerun_py/tests/api_sandbox/rerun_draft/catalog.py

```diff
@@ -242,7 +249,8 @@ def segment_table(
         view = DatasetView(self._inner, _LazyDatasetState())
         return view.segment_table(join_meta=join_meta, join_key=join_key)
 
-    def manifest(self) -> Any:
+    # What is this? What is it for?
```

Comment:
the dataset manifest, aka a table describing the content of a dataset where each row is a layer

Reply:
changed to return a `datafusion.DataFrame` for consistency in:
- https://github.com/rerun-io/rerun/pull/11969